### PR TITLE
feat: pasword reset flow

### LINKED ISSUE #458: Password reset flow
- [ ] Add `https://qri.cloud/signin/forgot` route with username or text input
- [ ] Ensure that the necessary endpoint exists
- [ ] Design email notification
- [ ] Set up email notification with template
- [ ] Confirm the entire flow from clicking "Forgot Password" in the sign up or login modals, receive email with link, click link, go to `signin/reset`, enter new password, see success message

### PR DESCRIPTION
**Pull request by boandriy**
pasword reset flow

closes #458

### CODE REVIEW

File: src/features/session/PasswordReset.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const handleSubmitClick = () => {
+    setInputError('')
+    if (password !== rePassword) {
+      setInputError("Passwords don't match")
+      return
+    }
+    if (password.length < 8) {
+      setInputError('Password must be at least 8 characters')
+      return
+    }
```
Comment (ramfox) [inline src/features/session/PasswordReset.tsx]: we have a `validatePassword` function used in the `SignUpModel`, that we should use here

File: src/features/session/ForgotPassword.tsx [hunk 2]
```diff
@@ -1,8 +1,93 @@
...
+  }
 
-const ForgotPassword: React.FC<any> = () => {
   return (
-    <h1>ForgotPassword</h1>
+    <div className='flex flex-col min-h-full bg-qrigray-100'>
+      <NavBar />
+      <PasswordForgotBgImage />
+      <ContentBox className='absolute top-60 left-12 md:left-64 z-30 ' paddingClassName='p-11'>
+        <h1 style={{lineHeight: '45px', width: 276}} className='font-extrabold text-3xl w-60 leading-10 mb-8'>{sent ? 'Password Sent!' :
```
Comment (ramfox) [inline src/features/session/ForgotPassword.tsx]: We say "Password Sent!" here, but we aren't actually sending the password over right? Should this instead say something like "Password Reset Sent!" @chriswhong

**Review by ramfox — APPROVED** [review]
I cannot check this flow locally, but if it has the ok from @Arqu, then LGTM!

One or two nit picks.

**Review by Arqu — APPROVED** [review]

File: src/features/session/ForgotPassword.tsx (hunk 2 above)
Comment (chriswhong) [reply src/features/session/ForgotPassword.tsx]: Let's say "You'll get an email with a link to reset your password shortly."

**Review by chriswhong — APPROVED** [review]
LGTM!

**Pull request merged**